src/sidebar/helpers/annotation-metadata.js

```diff
@@ -6,25 +6,34 @@
 /** @typedef {import('../../types/api').TextPositionSelector} TextPositionSelector */
 /** @typedef {import('../../types/api').TextQuoteSelector} TextQuoteSelector */
 
-/** Extract a URI, domain and title from the given domain model object.
```

Comment:
The terminology "domain model object" is not something we use any more in the client. We just say "an annotation", "a group" etc.